core/domain/suggestion_registry.py

```diff
@@ -451,7 +451,8 @@ def validate(self):
             None, state_domain.State.from_dict(
                 self.change.question_dict['question_state_data']),
             self.change.question_dict['question_state_data_schema_version'],
-            self.change.question_dict['language_code'], None)
+            self.change.question_dict['language_code'], None,
+            self.change.question_dict['linked_skill_ids'])
```

Comment:
This might need frontend changes too. It's not enabled (hidden behind the enable new structures players). It's been a while since we tested this flow, so I suspect there might some other bugs. If you like I can make a follow up PR with the frontend changes. Thanks!

Reply:
It does? I didn't have to make any frontend changes for the question editor and so I thought it would be the same for others. I don't really know much about the suggestion flow so if you have the time can you make those frontend changes? If that isn't possible, I can chat with you and understand what's happening with the suggestion flow and make the changes in a follow up.